all-services.jenkinsfile

```diff
@@ -48,60 +50,54 @@ pipeline
                 }
             }
         }
-        // It turns out that trying to do too many things in parallel here doesn't really save any time at all, the total running time was almost exactly the same - 2h17m for both!
-        // But the advantage of running these sequentially is that it's not as resource-demanding on the server as the parallel version is.
-        // stage("Generate supporting files and content")
-        // {
-        //     parallel
-        //     {
-                stage("analysis-core")
+
+        stage("analysis-core")
+        {
+            steps
+            {
+                dir("analysis-core")
                 {
-                    steps
+                    script
                     {
-                        dir("analysis-core")
-                        {
-                            script
-                            {
-                                docker.build("reactome/analysis-core:${env.RELEASE_VERSION} --build-arg RELEASE_VERSION=${env.RELEASE_VERSION} -f analysis-core.dockerfile")
-                            }
-                        }
+                        docker.build("reactome/analysis-core:${env.RELEASE_VERSION} --build-arg RELEASE_VERSION=${env.RELEASE_VERSION} -f analysis-core.dockerfile")
                     }
                 }
-                stage("fireworks-generator")
+            }
+        }
+        stage("fireworks-generator")
```

Comment:
The repo that is actually run is 'fireworks-layout' in Release. It looks like you might be not actually running the repo here, but rather in a directory named 'fireworks-generator' that is created, so it doesn't matter. Just wanted to say